### DISCUSSION TITLE
Text augmentation during training

### DISCUSSION THREAD

**Discussion by jn2clark** [mlfoundations/open_clip#542] [Announcements]
Has anyone successfully utilized augmentation on the text side of things? I would be interested to know what transformations worked and did not work.

**Comment by mitchellnw** [comment]
I don't think we've tried it here but check out the recent RECLIP paper

**Comment by vturrisi** [comment]
There's also a [paper](https://arxiv.org/abs/2305.08675) by Enrico (@DonkeyShot21) that uses text augmentations.

**Reply by DonkeyShot21** [reply]
Code will be out soonish. We don't do anything super fancy text-wise but tends to improve performance.

**Reply by jn2clark** [reply]
Thanks!
